src/driver/odoc_driver.ml

```diff
@@ -666,83 +677,51 @@ let run libs verbose packages_dir odoc_dir odocl_dir html_dir stats nb_workers
   Format.eprintf "Total time: %f@.%!" (Stats.total_time ());
   if stats then Stats.bench_results html_dir
 
-let fpath_arg =
-  let print ppf v = Fpath.pp ppf v in
-  Arg.conv (Fpath.of_string, print)
+module Voodoo_mode = struct
+  let run package_name = run (Voodoo { package_name })
 
-let odoc_dir =
-  let doc = "Directory in which the intermediate odoc files go" in
-  Arg.(value & opt fpath_arg (Fpath.v "_odoc/") & info [ "odoc-dir" ] ~doc)
+  let package_name =
+    let doc = "Name of package to process with voodoo" in
+    Arg.(value & pos 0 string "" & info [] ~doc)
 
-let odocl_dir =
-  let doc = "Directory in which the intermediate odocl files go" in
-  Arg.(value & opt (some fpath_arg) None & info [ "odocl-dir" ] ~doc)
+  let cmd =
+    let doc = "Process output from voodoo-prep" in
+    let info = Cmd.info "voodoo" ~doc in
+    Cmd.v info Term.(const run $ package_name $ Common_args.term)
+end
 
-let html_dir =
-  let doc = "Directory in which the generated HTML files go" in
-  Arg.(value & opt fpath_arg (Fpath.v "_html/") & info [ "html-dir" ] ~doc)
+module Dune_mode = struct
+  let run path = run (Dune { path })
 
-let packages =
-  (* TODO: Is it package or library? *)
-  let doc = "The packages to document" in
-  Arg.(value & opt_all string [] & info [ "p" ] ~doc)
+  let dune_style =
+    Arg.(value & pos 0 Common_args.fpath_arg (Fpath.v ".") & info [])
 
-let verbose =
-  let doc = "Enable verbose output" in
-  Arg.(value & flag & info [ "v"; "verbose" ] ~doc)
+  let cmd =
+    let doc = "Dune mode" in
```

Comment:
We can add a little bit more info here! Especially since it might not be obvious that it works on `_build/default/` and not on `_build/install/default/`.

```suggestion
    let doc =
      "Dune mode, which builds the documentation of the local libraries of a \
       dune project."
    in
```